Makefile

```diff
@@ -14,14 +14,20 @@ help:
 	@echo "Make tasks for installation and deployment. Refer to the Makefile content to know more."
 
 logs:  # `make type=access` or `make type=errors` for displaying logs.
-	@echo "> Reading ${type} logs."
-	${remote} "${goto_src} && tail -f logs/${type}.log"
+	@${remote} "tail -f ~/log/preprod.log"
+
+flush-logs:
+	@${remote} "echo '' > ~/log/preprod.log"
+	@echo "Log file emptied!"
 
 deploy:
 	@echo "> Fetching master branch and updating sources."
 	${remote} "${goto_src} && git fetch origin ${branch} && git checkout ${branch} && git reset --hard FETCH_HEAD"
 	${remote} "${goto_src} && pip install -r requirements.txt"
-	${remote} "pkill flask && source ~/cafebabel.com/venv/bin/activate && FLASK_APP=prod flask run >> ~/log/preprod-$(date +%Y%m%d).log 2>&1 &"
+	${remote} "${goto_src} && pkill gunicorn; \
+		gunicorn --daemon -b 0.0.0.0:5000 prod:app \
+		--error-logfile ~/log/preprod.log --access-logfile ~/log/preprod.log"
```

Comment:
Don't you want to have `preprod-access.log` and `preprod-error.log`?

Reply:
I logged it all in one single file as we use this for live viewing errors and there's not much visit here.
If you see any point to get it splitted I can make the adapting @davidbgk.